commands/gist.go

```diff
@@ -72,33 +68,27 @@ func init() {
 	CmdRunner.Use(cmdGist)
 }
 
-func getGist(gh *github.Client, id string, filename string, emitJson bool) error {
+func getGist(gh *github.Client, id string, filename string) error {
 	gist, err := gh.FetchGist(id)
 	if err != nil {
 		return err
 	}
 
-	if len(gist.Files) > 1 && !emitJson && filename == "" {
-		return fmt.Errorf("There are multiple files, you must specify one, or use --json")
+	if len(gist.Files) > 1 && filename == "" {
+		filenames := []string{}
+		for name := range gist.Files {
+			filenames = append(filenames, name)
+		}
+		return fmt.Errorf("the gist contains multiple files, you must specify one:\n%s", strings.Join(filenames, "\n"))
```

Comment:
should we sort these for consistent output and also to make the tests more exact?